src/SingleBookingPage/BookingDetail.js

```diff
@@ -82,7 +82,7 @@ class BookingDetail extends React.Component<Props> {
             <button className="manage-booking">Manage my booking</button>
           </a>
         </div>
-        {isUrgent && <Contact />}
+        {isUrgent && <Contact booking={booking} />}
```

Comment:
I would suggest to use different name for prop `<Contact booking={booking} />` looks suspicious at the first sight.

Reply:
e.g. info or even default data

or you could create fragment on `BookingContactDetails` and then `<Contact contacts={booking.contactDetails} />`

Reply:
Thanks for this suggestion, I changed the prop name to `info`, but im not sure if I get the second part of you suggestion. If you could make a quick example that would be great 🤔